config.py

```diff
@@ -72,15 +41,63 @@
             'External links',
             'Further reading',
             'Notes',
+            'Additional sources',
+            'Sources',
```

Comment:
Since we have a dictionary of section names as part of the model, could we skip all sections that are not in the dictionary, rather than configuring here?

Reply:
I think we need to keep them because `sections_to_skip` here specifies sections that we don't want to parse the contents and predict scores for sentences.

As I understand it, for sections that are not in the model dictionary `section_dict_en.pck` we may still want to parse contents and extract sentences to predict scores, although the section name we had never seen during the model training phase, so they are not in the dictionary.

Reply:
Oh, yeah, I actually don't know if we can end up with predictions for sentences not in the dictionary or not. Another question for @mirrys I guess.

Reply:
Hi, as @AikoChou said, it would be better to keep them. The model will asign an 'unkown' value for unknown sections, and proceed in analysing the text of the sentence.